lib/improver/utilities/cube_manipulation.py

```diff
@@ -835,6 +835,26 @@ def enforce_coordinate_ordering(
     return cube
 
 
+def enforce_float32_precision(input_cubes):
+    """Take input cube of any precision and convert to float32.
+
+    Args:
+        input_cubes (list of cubes or single cube):
```

Comment:
I think this should be as shown below, as list the brackets should contain a python type e.g. list, np.array etc:
```python
input_cubes (list):
```